libraries/AP_Motors/AP_MotorsMatrix.cpp

```diff
@@ -115,6 +119,12 @@ void AP_MotorsMatrix::output_to_motors()
             rc_write(i, motor_out[i]);
         }
     }
+
+    // plane outputs for Qmodes are setup here, and written to the HAL by the plane servos loop
+    SRV_Channels::set_output_scaled(SRV_Channel::k_aileron, -_yaw_in*4500.0f);
+    SRV_Channels::set_output_scaled(SRV_Channel::k_elevator, _pitch_in*4500.0f);
+    SRV_Channels::set_output_scaled(SRV_Channel::k_rudder, _roll_in*4500.0f);
+    SRV_Channels::set_output_pwm(SRV_Channel::k_throttle, throttle_pwm);
```

Comment:
@IamPete1 is is necessary to set the throttle channel here? I think all motor ouputs are sent to channels k_motorN in this approach.

Reply:
I think your right, probably don't need the throttle here any more, I just copied the code from the tailsitter motors bit, but its probably not needed there any more either since I moved all the mixers